src/pool/pool_scalable.c

```diff
@@ -167,6 +167,7 @@ static void tbb_pool_finalize(void *pool) {
     pthread_once(&tbb_is_initialized, load_tbb_symbols);
```

Comment:
Not really connected to this PR but why do we initialize symbols in finalize? By the time someone calls finalize, initialize() would have been called and it would initialize the symbols, right?

Reply:
I guess that this is due to a later call to pool_destroy(). Do you think we should not call pthread_once() here? IMO it could stay.